### PR TITLE
Add Make Say effect

### PR DESCRIPTION
**Pull request by Blueyescat**
Target Minecraft versions: Any
Requirements: No
Related issues: https://github.com/SkriptLang/Skript/issues/1291

Description:
Adds an effect that forces players to say messages in the chat.

### CODE REVIEW

File: src/main/java/ch/njol/skript/effects/EffMakeSay.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+@Name("Make Say")
+@Description("Forces a player to send a message to the chat. If the message starts with a slash it will force the player to use command.")
+@Examples({"make the player say \"Hello.\"", "force all players to send the message \"I love this server\""})
+@Since("INSERT VERSION")
+public class EffMakeSay extends Effect {
+
+	static {
+		Skript.registerEffect(EffMakeSay.class,
+				"make %players% (say|send [the] message[s]) %strings%",
+				"force %players% to (say|send [the] message[s]) %strings%");
```
Comment (Pikachu920) [inline src/main/java/ch/njol/skript/effects/EffMakeSay.java]: why not just (make|force) instead of repeating the whole syntax?

Comment (Pikachu920) [reply src/main/java/ch/njol/skript/effects/EffMakeSay.java]: ah i missed that difference

**Comment by TheLimeGlass** [comment]
Just extend the force command effect.

**Comment by Blueyescat** [comment]
@TheLimeGlass What? It should use the `chat` method.

**Comment by TheLimeGlass** [comment]
Not the method you idiot, the actual syntax class is what I'm referring too.

**Comment by Blueyescat** [comment]
Why.

**Comment by Snow-Pyon** [comment]
Thanks for the contribution!

**Pull request merged**